man/dcp.1

```diff
@@ -104,7 +104,7 @@ that each symbolic link refers to.
 .B \-P, \-\-no\-dereference
 Do not follow symbolic links in source paths. Effectviely allows
 symbolic links to be copied when the link target is not valid
-or there is not permission to read the link’s target.
+or there is not permission to read the link\(aqs target.
```

Comment:
Again

Reply:
Hi @gonsie are you certain the `\(aq` is incorrect?  See my comments on https://github.com/hpc/mpifileutils/issues/591.  When I look at this generated man/dcp.1 file with the terminal `man` utility on oslic, it renders correctly.